reporter/src/main/kotlin/Statistics.kt

```diff
@@ -155,12 +158,14 @@ data class LicenseStatistics(
     /**
      * All declared licenses, mapped to the number of [Project]s and [Package]s they are declared in.
      */
-    val declared: SortedMap<String, Int>,
+    @JsonSerialize(converter = ComparableSortedMapConverter::class)
```

Comment:
You should be able to just use `@JsonPropertyOrder(alphabetic = true)` instead (also see the `CopyrightStatementsProcessor`). Similar below.

Reply:
Nice, that's a great hint! FYI: Above approach didn't work because @JsonValue in `Identifier` became ignored, maybe due to type erasure.